communicator/main.go

```diff
@@ -9,15 +9,18 @@ import (
 const port = 4000
 
 func main() {
-
 	fmt.Printf("Starting communicator at :%d\n", port)
+	err := LoadQuestions()
```

Comment:
the function is effectful, make it return a list of questions and assign it to a variable